src/pages/survey.js

```diff
@@ -0,0 +1,58 @@
+import { IconButton} from '@chakra-ui/react'
+import { Button } from "@chakra-ui/react";
+import {BsEmojiFrownFill, BsFillEmojiNeutralFill, BsFillEmojiSmileFill } from "react-icons/bs"
+
+
+export const Survey = () => {
+
+
+
+return (
+<div className="survey">
+    <div className="">
+        <h2>Survey</h2>
+        
+
+
+        <div className="survey-text">
```

Comment:
Can we swap the tags in this to the chakra components?

Reply:
sure, we can do that.